.github/workflows/create-sprint-planning-meeting.yaml

```
@@ -11,7 +11,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: JasonEtco/create-an-issue@v2
+    - uses: actions/setup-python@v2
+    - name: Calculate if we're on a 2-week cycle
+      run: python datetime.py
+      id: is-two-weeks
+    - name: Open a new issue
+      if: steps.is-two-weeks.IS_TWO_WEEKS == 'True'
```

```suggestion
      if: ${{ steps.is-two-weeks.IS_TWO_WEEKS }} == 'True'
```

Eep, my comment got mangled (on my phone, sorry!), this is the suggestion I meant to send 

Eep, my comment got ma